core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -894,8 +894,8 @@ package object config {
       .checkValue(_ > 0, "The max no. of blocks in flight cannot be non-positive.")
       .createWithDefault(Int.MaxValue)
 
-  private[spark] val MAX_REMOTE_BLOCK_SIZE_FETCH_TO_MEM =
-    ConfigBuilder("spark.maxRemoteBlockSizeFetchToMem")
+  private[spark] val NETWORK_MAX_REMOTE_BLOCK_SIZE_FETCH_TO_MEM =
```

Comment:
we can keep using `MAX_REMOTE_BLOCK_SIZE_FETCH_TO_MEM`

Reply:
Thanks, done in 8f7540e.